### PR TITLE
Task4

### PR DESCRIPTION
**Pull request by grafgeest**
[task4] Added directives and pipes

### CODE REVIEW

File: src/app/core/pipes/filter-by.pipe.spec.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { FilterByPipe } from './filter-by.pipe';
+
+describe('FilterByPipe', () => {
+  it('create an instance', () => {
+    const pipe = new FilterByPipe();
+    expect(pipe).toBeTruthy();
```
Comment (vnovikov-clgx) [inline src/app/core/pipes/filter-by.pipe.spec.ts]: You need to test what should be return, i mean if you pass 120.

File: src/app/core/pipes/order-by.pipe.spec.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { OrderByPipe } from './order-by.pipe';
+
+describe('OrderByPipe', () => {
+  it('create an instance', () => {
+    const pipe = new OrderByPipe();
+    expect(pipe).toBeTruthy();
```
Comment (vnovikov-clgx) [inline src/app/core/pipes/order-by.pipe.spec.ts]: You need test return value

File: src/app/core/pipes/filter-by.pipe.spec.ts (hunk 1 above)
Comment (grafgeest) [reply src/app/core/pipes/filter-by.pipe.spec.ts]: Fixed

File: src/app/core/pipes/order-by.pipe.spec.ts (hunk 2 above)
Comment (grafgeest) [reply src/app/core/pipes/order-by.pipe.spec.ts]: Fixed

**Pull request merged**